devicetypes/smartthings/fidure-2nd-generation-thermostat.src/fidure-2nd-generation-thermostat.groovy

```diff
@@ -0,0 +1,810 @@
+/**
+ *  Copyright 2016 SmartThings, Inc.
```

Comment:
I assume this should be "Copyright 2017 Fidure Corp."

Reply:
changed